core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserManagementErrorListener.java

```diff
@@ -205,7 +205,7 @@ public boolean onUpdatePermissionsOfRoleFailure(String errorCode, String errorMe
      * @return true if the handing succeeded.
      * @throws UserStoreException Exception that will be thrown during the execution of the method.
      */
-    boolean onGetPaginatedUserListFailure(String errorCode, String errorMessage, String claim, String claimValue,
+    public boolean onGetPaginatedUserListFailure(String errorCode, String errorMessage, String claim, String claimValue,
```

Comment:
Since the default method is in the interface, do we need this method overridden here?

Reply:
As the implementation is the same here we can remove them as well. So shall we remove all these moved methods from the Abstract listener level?